.pre-commit-config.yaml

```diff
@@ -1,3 +1,4 @@
+minimum_pre_commit_version: '2.9.0'
```

Comment:
let's make this `2.9.2`, as the release note mentions

> Fix default value for types_or so symlink and directory can be matched

Reply:
Fixed!